node/node.go

```diff
@@ -1,7 +1,5 @@
 package node
 
 type Node struct {
-	CPUs   int    `json:"cpus" yaml:"cpus"`
-	ID     string `json:"id" yaml:"id"`
-	Memory string `json:"memory" yaml:"memory"`
+	ID string `json:"id" yaml:"id"`
```

Comment:
We also need a node ID on AWS for scaling workers up and down. @oponder is looking at adding this to cluster-service since it already generates the cluster ID.

See https://github.com/giantswarm/giantswarm/issues/1461#issuecomment-309712830